Airwallex/Card/Internal/AWXSecurityService.m

```diff
@@ -47,25 +45,24 @@ - (instancetype)init
     return self;
 }
 
-- (void)doProfile:(NSString *)intentId
-       completion:(void(^)(NSString * _Nullable))completion
-{
+- (void)doProfile:(NSString *)intentId completion:(void (^)(NSString *_Nullable))completion {
 #if TARGET_OS_SIMULATOR
     completion([UIDevice currentDevice].identifierForVendor.UUIDString);
 #else
     double timestamp = [[NSDate date] timeIntervalSince1970] * 1000;
     NSString *fraudSessionId = [NSString stringWithFormat:@"%@%.0f", intentId, timestamp];
-    [self.profiling profileDeviceUsing:@{RLTMXSessionID: fraudSessionId} callbackBlock:^(NSDictionary *result) {
-        RLTMXStatusCode statusCode = [[result valueForKey:RLTMXProfileStatus] integerValue];
-        dispatch_async(dispatch_get_main_queue(), ^{
-            if (statusCode == RLTMXStatusCodeOk) {
-                NSString *sessionId = result[RLTMXSessionID];
-                completion(sessionId ?: @"");
-                return;
-            }
-            completion(@"");
-        });
-    }];
+    [self.profiling profileDeviceUsing:@{RLTMXSessionID: fraudSessionId}
+                         callbackBlock:^(NSDictionary *result) {
+                             RLTMXStatusCode statusCode = [[result valueForKey:RLTMXProfileStatus] integerValue];
+                             dispatch_async(dispatch_get_main_queue(), ^{
+                                 if (statusCode == RLTMXStatusCodeOk) {
+                                     NSString *sessionId = result[RLTMXSessionID];
+                                     completion(sessionId ?: @"");
+                                     return;
+                                 }
+                                 completion(@"");
+                             });
+                         }];
```

Comment:
This is a bit weird as well. I'd expect it will move the ending ] to the next line 🙈

Reply:
I think that's the expected formatting - kind of makes sense since the opening brace doesn't have a newline after it.